scenarios/3ctrl_xcpt_xceph/infra.yaml

```diff
@@ -0,0 +1,151 @@
+          - cloud::telemetry::collector
+          - cloud::telemetry::alarmevaluator
+          - cloud::telemetry::alarmnotifier
+          - cloud::telemetry::notification
+          - cloud::network::controller
+          - cloud::orchestration::api
+          - cloud::orchestration::engine
+          - cloud::telemetry::centralagent
+      5:
+        roles:
+          - cloud::network::dhcp
+          - cloud::network::metadata
+          - cloud::network::lbaas
+          - cloud::network::l3
+    serverspec_config:
+      ks_user_name: ks_admin_tenant
+      ks_user_password: ks_admin_password
+      ks_tenant_name: ks_admin_tenant
+      vip_public: vip_public_fqdn
+      endpoint_protocol: endpoint_protocol
+      virt_type: libvirt_type
+      neutron_password: ks_neutron_password
+      neutron_endpoint: vip_public_fqdn
+      neutron_protocol: ks_neutron_admin_proto
+      neutron_type_drivers: neutron_type_drivers
+      neutron_mechanism_drivers: neutron_mechanism_drivers
+      neutron_tunnel_id_ranges: neutron_tunnel_id_ranges
+      neutron_firewall_driver: neutron_firewall_driver
+      syslog_server: syslog_server
+      syslog_port: syslog_port
+      puppetdb_server: puppetdb_server
+  compute:
+    arity: 1+n
+    edeploy: openstack-full
+    steps:
+      4:
+        roles:
+          - cloud::compute::hypervisor
+    serverspec_config:
+      ks_user_name: ks_admin_tenant
+      ks_user_password: ks_admin_password
+      ks_tenant_name: ks_admin_tenant
+      vip_public: vip_public_fqdn
+      endpoint_protocol: endpoint_protocol
+      virt_type: libvirt_type
+      neutron_password: ks_neutron_password
+      neutron_endpoint: vip_public_fqdn
+      neutron_protocol: ks_neutron_admin_proto
+      neutron_type_drivers: neutron_type_drivers
+      neutron_mechanism_drivers: neutron_mechanism_drivers
+      neutron_tunnel_id_ranges: neutron_tunnel_id_ranges
+      neutron_firewall_driver: neutron_firewall_driver
+      syslog_server: syslog_server
+      syslog_port: syslog_port
+      puppetdb_server: puppetdb_server
+  storage:
+    arity: 3+1n
+    edeploy: openstack-full
+    steps:
+      2:
+        roles:
+          - cloud::storage::rbd::osd
+    serverspec_config:
+      ks_user_name: ks_admin_tenant
+      ks_user_password: ks_admin_password
+      ks_tenant_name: ks_admin_tenant
+      vip_public: vip_public_fqdn
+      endpoint_protocol: endpoint_protocol
+      virt_type: libvirt_type
+      neutron_password: ks_neutron_password
+      neutron_endpoint: vip_public_fqdn
+      neutron_protocol: ks_neutron_admin_proto
+      neutron_type_drivers: neutron_type_drivers
+      neutron_mechanism_drivers: neutron_mechanism_drivers
+      neutron_tunnel_id_ranges: neutron_tunnel_id_ranges
+      neutron_firewall_driver: neutron_firewall_driver
+      syslog_server: syslog_server
+      syslog_port: syslog_port
+      puppetdb_server: puppetdb_server
+
+ordered_profiles:
+  - install-server
+  - load-balancer
```

Comment:
@clopnis You need to remove this line because you have the load-balancer on the controller